tools/ci/check_grep2.py

```diff
@@ -101,10 +101,9 @@ def check_to_chats_have_a_user_arguement(lines):
         if TO_CHAT_WITH_NO_USER_ARG_RE.search(line):
             return Failure(idx + 1, "Changed files contains a to_chat() procedure without a user argument.")
 
-CONDITIONAL_LEADING_SPACE = re.compile(r"(if|for|while|switch)\s+(\(.*?\))") # checks for "if (thing)", replace with $1$2
-CONDITIONAL_BEGINNING_SPACE = re.compile(r"(if|for|while|switch)(\(.+) \)") # checks for "if( thing)", replace with $1$2)
-CONDITIONAL_ENDING_SPACE = re.compile(r"(if|for|while|switch)\( (.+\))") # checks for "if(thing )", replace with $1($2
-CONDITIONAL_INFIX_NOT_SPACE = re.compile(r"(if)\(! (.+\))") # checks for "if(! thing)", replace with $1(!$2
```

Comment:
No longer required?

Reply:
Moved to be a part of one of the other checks, as `(!)?`